package.json

```diff
@@ -33,20 +33,20 @@
     "contrib/"
   ],
   "devDependencies": {
-    "@esotericsoftware/spine-core": "4.1.36",
-    "@playcanvas/eslint-config": "^1.5.0",
-    "esbuild": "^0.19.0",
-    "eslint": "^8.46.0",
-    "grunt": "^1.6.1",
-    "grunt-contrib-concat": "^2.1.0",
-    "grunt-contrib-uglify": "^5.2.2",
+    "@playcanvas/eslint-config": "^1.7.0",
+    "@rollup/plugin-alias": "^5.0.0",
+    "@rollup/plugin-commonjs": "^25.0.4",
+    "@rollup/plugin-node-resolve": "^15.2.1",
+    "@rollup/plugin-terser": "^0.4.3",
+    "esbuild": "^0.19.2",
```

Comment:
Oh, actually, is `esbuild` needed any more?

Reply:
That is part of the build system for consuming vendor typescript.

Reply:
Can you point me at where it's invoked? I don't see it in the repo....

Reply:
Future bits, I updated the notes up top.  Might be worth pulling in the official "@rollup/plugin-typescript" now so we don't need to touch the build system in other PRs ?

Reply:
If it's needed for future bits, I would include it in a future PR. And at that point, we can debate whether it's required. But if the current state of the repo doesn't require `esbuild`, I would remove it. Again, this is me having a strong preference to keep PRs minimal, complete and atomic.

Reply:
Your call.  Build changes in one build PR or when its used.  We know we'll be consuming typescript next, that's what the vendors code really is.

Reply:
I definitely prefer 'when it's used'. 😄

Reply:
Then it goes.  Completed in 40db1ce